### PR TITLE
MUI and dependencies Version Upgrade

### PR DESCRIPTION
**Pull request by abdullai-t**
####  Summary / Highlights
This pr contains an upgrade of material ui to v5 and its dependencies. For compatibility sake,  Reactjs has also been upgraded to v18.  Follow link for [MUI v5](https://mui.com/material-ui/getting-started/overview/) documentation and new imports.

#### Details (Give details about what this PR accomplishes, include any screenshots etc)

https://user-images.githubusercontent.com/42780120/215614178-aff4c08b-486d-4a76-8683-3cd56655906d.mp4

#### Testing Steps (Provide details on how your changes can be tested)

- Delete node_modules and reinstall all packages with `npm i `

#### Requirements (place an `x` in each `[ ]`)
* [x] I've read and understood the [Contributing Guidelines](/blob/main/.github/contributing.md) and have done my best effort to follow them.
* [x] I've tested my changes manually.
* [ ] I've added unit tests to my PR.

##### Transparency ([Project board](https://github.com/orgs/massenergize/projects/7/views/2))
* [x] I've given my PR a meaningful title.
* [x] I linked this PR to the relevant issue.
* [x] I moved the linked issue from the "Sprint backlog" to "In progress" when I started this.
* [ ] I moved the linked issue to "QA Verification" now that it is ready to merge.

### CODE REVIEW

File: app/components/CardPaper/cardStyle-jss.js [hunk 1]
```diff
@@ -1,64 +1,65 @@
-import pink from '@material-ui/core/colors/pink';
-import lightGreen from '@material-ui/core/colors/lightGreen';
-import dark from '@material-ui/core/colors/blueGrey';
-import { fade, darken } from '@material-ui/core/styles/colorManipulator';
+import pink from '@mui/material/colors/pink';
+import lightGreen from '@mui/material/colors/lightGreen';
+import dark from "@mui/material/colors/blueGrey";
+darken
```
Comment (BradHN1) [inline app/components/CardPaper/cardStyle-jss.js:4]: this line a mistake?

**Review by BradHN1 — APPROVED** [review]
Fantastic job!  It works fine!

**Comment by abdullai-t** [comment]
@BradHN1 - After hours of trying to fix the bug causing the page to go blank after submitting a form, I finally fixed it 😂. The bug was caused by the Snackbar component used in the form generator to show messages to users after a success or failure during form submission. Here is a demo of the forms now working.

https://user-images.githubusercontent.com/42780120/216337132-80416912-e9c0-4e46-a0b9-ce344d962a3e.mp4

**Review by BradHN1 — APPROVED** [review]
Excellent job!  Time to merge to Dev

**Pull request merged**